src/pages/_app.tsx

```diff
@@ -39,10 +40,25 @@ type AppPropsWithLayout = AppProps & {
   Component: NextPageWithLayout;
 };
 
+const hostUrl =
+  process.env.NODE_ENV === 'development'
+    ? 'localhost:3000'
+    : process.env.VERCEL_URL;
+
 const SEO: DefaultSeoProps = {
   titleTemplate: '%s | Vegan Hacktivists',
   openGraph: {
     url: 'https://veganhacktivists.org',
+    description:
+      "We're a diverse community of passionate vegan activists from all around the world, volunteering our time and skills towards the animal protection movement. We build free technology and offer free services with the goal of reducing or stopping the mass amounts of suffering caused by factory farming.",
+    images: [
+      {
+        url: `${hostUrl}${logoBig.src}`,
```

Comment:
Changed this URL to a full absolute URL (it was a relative URL before). Per this comment:
https://github.com/garmeeh/next-seo/issues/14#issuecomment-503715239